streams/src/test/java/org/apache/kafka/streams/state/internals/TimestampedWindowStoreBuilderTest.java

```diff
@@ -223,38 +208,9 @@ public void shouldThrowNullPointerIfInnerIsNull() {
         assertThrows(NullPointerException.class, () -> new TimestampedWindowStoreBuilder<>(null, Serdes.String(), Serdes.String(), new MockTime()));
     }
 
-    @Test
-    public void shouldThrowNullPointerIfKeySerdeIsNull() {
-        assertThrows(NullPointerException.class, () -> new TimestampedWindowStoreBuilder<>(supplier, null, Serdes.String(), new MockTime()));
-    }
-
-    @Test
-    public void shouldThrowNullPointerIfValueSerdeIsNull() {
-        assertThrows(NullPointerException.class, () -> new TimestampedWindowStoreBuilder<>(supplier, Serdes.String(), null, new MockTime()));
-    }
-
     @Test
     public void shouldThrowNullPointerIfTimeIsNull() {
         assertThrows(NullPointerException.class, () -> new TimestampedWindowStoreBuilder<>(supplier, Serdes.String(), Serdes.String(), null));
     }
 
-    @Test
-    public void shouldThrowNullPointerIfMetricsScopeIsNull() {
```

Comment:
Why did we remove this test @clolov? It seems like we do indeed expect a NPE for metrics scope at `Objects.requireNonNull(storeSupplier.metricsScope(), "storeSupplier's metricsScope can't be null");` in `TimestampedWindowStoreBuilder.java`

Reply:
This is a fair point, the problem here is that the setUp method mocked the supplier.metricScope to return a non-null value. I have remedied this since, thank you for the thorough review!